static/src/javascripts/projects/common/modules/commercial/badges.js

```diff
@@ -69,12 +69,12 @@ define([
 
             return new Promise(function (resolve) {
                 idleFastdom.write(function () {
-                    var placeholder = $('.js-badge-placeholder', container);
+                    var placeholder = $('.js-badge-placeholder', item);
 
                     if (placeholder.length) {
                         placeholder.replaceWith($adSlot);
                     } else {
-                        $('.js-container__header', container).after($adSlot);
+                        $(opts.fallback, item).after($adSlot);
```

Comment:
Small detail but as `fallback` is mandatory, let's make it explicit by making it a bona fide function parameter.